doc/news/new_opmath.rst

```diff
@@ -36,6 +36,12 @@ Summary of the update
     If it still does, it likely only requires some minor changes. For that, see the :ref:`Troubleshooting_opmath` section.
     You can still opt-out and run legacy code via ``qml.operation.disable_new_opmath()``.
 
+.. warning::
+
+    In PennyLane v0.39, legacy operator arithmetic is deprecated and will be removed in v0.40. Check out the functions
+    :func:`~pennylane.operation.convert_to_opmath` and :func:`~pennylane.operation.convert_to_H` to convert legacy operators
+    into those using the new operator arithmetic.

```

Comment:
I did not deprecate these functions because they're the only way users can easily convert legacy ops into new ops without having to do something like

```python
new_op = qml.Hamiltonian(old_op.coeffs, old_op.ops)`
```